backoffice/src/Views/Email/Index.cshtml

```diff
@@ -3,6 +3,20 @@
     ViewData["Title"] = $"Emails ({Model.EmailList.Count})";
 }
 
+@if (TempData.TryGetValue("SuccessMessage", out var successMessage))
```

Comment:
I'm thinking once we have them renamed as "SuccessMessage" and "ErrorMessage" these banners can be moved out into a partial view we can include on most views. 

What are your thoughts on that?

Reply:
Added a partial for this